src/append/rolling_file/policy/compound/roll/fixed_window.rs

```diff
@@ -125,7 +132,7 @@ impl Roll for FixedWindowRoller {
             move_file(&src, &dst)?;
         }
 
-        self.compression.compress(file, &dst_0).map_err(Into::into)
+        self.compression.compress(current_log, &dst_0).map_err(Into::into)
```

Comment:
Compression is still turned on?

Reply:
`self.compression.compress()` actually renames the file.   THEN it compresses if compression is turned on.   Since that is a feature flag, I left it alone for now.